boards/hip-badge/include/board.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2023 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_hip_badge
+ * @brief       Board definitions for the HiP Badge
+ * @{
+ *
+ * @file
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include <stdint.h>
+
+/**
+ * @name    Button pin definitions
+ * @{
+ */
+
+/**
+ * @brief   Tact A GPIO pin definition
+ */
+#define BTN0_PIN        GPIO2
+
+/**
+ * @brief    button GPIO mode definition
+ *
+ * Since the GPIO of the button is pulled up with an external resistor, the
+ * mode for the GPIO pin has to be GPIO_IN.
+ */
+#define BTN0_MODE       GPIO_IN_PU
```

Comment:
What is correct, is it pulled up with an external resistor or with an internal resistor?
```suggestion
 */
#define BTN0_MODE       GPIO_IN_PU
```

Reply:
There is an external pull-up

![image](https://user-images.githubusercontent.com/1301112/211061066-f2115dc2-4ba8-41b6-bafa-e41093613b86.png)

Reply:
Ok, but then the mode should be `GPIO_IN`.